src/app/lib/config/services/afaanoromoo.js

```diff
@@ -0,0 +1,159 @@
+import { C_POSTBOX } from '@bbc/psammead-styles/colours';
```

Comment:
These all broadly seem fine (for a tester to check really), they need importing/adding into `config/services/index.js` and `config/services/loadableConfig.js` then I think we can view these service routes locally.